modules/ROOT/pages/aurads/importing-data/load-csv.adoc

```diff
@@ -14,15 +14,15 @@ In this example we will load three CSV files:
 * `people.csv`: a list of actors with their year of birth
 * `actors.csv`: a list of acting roles, where actors are matched with the movies they had a role in
 
-WARNING: *Warning*: The `LOAD CSV` command is built to handle small to medium-sized data sets, such as anything up to 10 million nodes and relationships. You should avoid using this command for any data sets exceeding this limit.
+WARNING: The `LOAD CSV` command is built to handle small to medium-sized data sets, such as anything up to 10 million nodes and relationships. You should avoid using this command for any data sets exceeding this limit.
 
 include::partial$aurads/setup.adoc[]
 
 == Create constraints
 
 Adding constraints before loading any data usually improves data loading performance. In fact, besides adding an integrity check, a unique constraint adds an index on a property at the same time, so that `MATCH` and `MERGE` operations during loading are faster.
 
-WARNING: *Warning*: For best performance when using `MERGE` or `MATCH` with `LOAD CSV`, make sure an index or a unique constraint has been created on the property used for merging. For more information, follow the link:{neo4j-docs-base-uri}/getting-started/current/cypher-intro/load-csv/#_prepare_the_database[Import data] tutorial in the Cypher documentation.
+WARNING: For best performance when using `MERGE` or `MATCH` with `LOAD CSV`, make sure an index or a unique constraint has been created on the property used for merging. For more information, follow the link:{neo4j-docs-base-uri}/getting-started/current/cypher-intro/load-csv/#_prepare_the_database[Import data] tutorial in the Cypher documentation.
```

Comment:
The linked page does not really provide more information about constraints. Maybe link here as well/instead: https://neo4j.com/docs/cypher-manual/current/constraints/

Reply:
You're right, good call!